pkg/libinit/root_linux.go

```diff
@@ -196,6 +196,12 @@ func SetEnv() {
 	for k, v := range env {
 		os.Setenv(k, v)
 	}
+
+	// Backwards compatibility for the transition from uroot.nohwrng to
+	// UROOT_NOHWRNG=1 on kernel commandline.
+	if cmdline.ContainsFlag("uroot.nohwrng") {
+		os.Setenv("UROOT_NOHWRNG", "1")
```

Comment:
This seems unrelated to the purpose of this PR? Could this be a separate PR?

Reply:
It cannot be, sorry -- when I copied pkg/rand from u-root/u-root to u-root/uio, I removed "uroot.nohwrng" because it was the only dependency on pkg/cmdline. I could copy pkg/cmdline, but I'd rather make sure that u-root/uio builds and works on all platforms.

Hence, there has to be this little transition here. I'll add a deprecation warning.